packages/sonarwhal/src/lib/utils/resource-loader.ts

```diff
@@ -280,3 +284,65 @@ export const loadFormatter = (formatterId: string): IFormatter => {
 export const loadParser = (parserId: string): Parser => {
     return loadResource(parserId, TYPE.parser);
 };
+
+const searchNpmPackages = (searchTerm: string): Promise<Array<NpmPackage>> => {
+    return new Promise((resolve, reject) => {
+        const results = [];
+
+        const searchOptions = {
+            description: true,
+            excluded: [],
+            include: [searchTerm],
+            limit: 1000,
+            staleness: 900,
+            unicode: false
+        };
+
+        esearch(searchOptions)
+            .on('data', (data) => {
+                results.push(data);
+            })
+            .on('error', (err) => {
+                reject(err);
+            })
+            .on('end', () => {
+                resolve(results);
+            });
+    });
+};
+
+const loadNpm = () => {
+    return npmLoadAsync({ loaded: false });
+};
+
+export const getNpmPackages = async (searchTerm: string = 'sonarwhal'): Promise<Array<NpmPackage>> => {
+    await loadNpm();
+
+    return searchNpmPackages(searchTerm);
+};
+
+const filterPackages = (packages: Array<NpmPackage>, initTerm: string) => {
```

Comment:
Move this up so the `export`s are all together.

Also add description saying it filters by `startsWith`